set.mm

```diff
@@ -662622,39 +662662,34 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
   $( $j usage 'nn0mulcom' avoids 'ax-mulcom'; $)
 
   $( Lemma for ~ zmulcom .  (Contributed by SN, 25-Jan-2025.) $)
-  zmulcomlem $p |- ( ( ( A e. RR /\ ( 0 -R A ) e. NN ) /\ B e. NN ) ->
+  zmulcomlem $p |- ( ( ( A e. RR /\ ( 0 -R A ) e. NN ) /\ B e. NN0 ) ->
```

Comment:
It's strange that the proof becomes longer, and probably with "`( 0 -R A ) e. NN0`" it was worse which is why you didn't do it ?  That's probably a sign that earlier library lemmas are lacking.  That's just a remark, nothing blocking this PR.